test/e2e/framework.go

```diff
@@ -197,6 +200,12 @@ type TestOptions struct {
 	windowsVMs          string
 }
 
+type flowVisibilityTestOptions struct {
+	databaseURL      string
+	secureConnection bool
+	name             string
```

Comment:
this `name` field doesn't make sense in this struct

Reply:
Fixed. Thanks.